### ISSUE TITLE
Does this work with Universal Analytics?

### ISSUE THREAD

**Issue by jamesspittal** [joaolcorreia/Google-Analytics-PHP-cookie-parser#9]
I see in your example, you are using Google Analytics (`ga.js`) as opposed to Universal Analytics (`analytics.js`). Is this compatible with `analytics.js`? I'm guessing not because Universal Analytics uses the `_ga` cookie as opposed to `__utma` and `__utmz`

**Comment by akassover** [comment]
Nope. Universal Analytics keeps the data you're looking for locked up in Google's datacenter.

**Comment by jamesspittal** [comment]
Thought so. Thanks @akassover

**Comment by designorant** [comment]
You can use both `ga.js` and `analytics.js` at the same time. There's [plenty of information about that](http://stackoverflow.com/questions/17036884/how-to-use-both-ga-js-and-analytics-js) everywhere. Bear in mind that Google will kill it at some point, though.
